app/controllers/v0/sessions_controller.rb

```diff
@@ -82,7 +82,7 @@ def saml_user
 
     def create_saml_user
       user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? || user.gender.nil?
+      user.mvi # caches mvi data
```

Comment:
@aud I'll move it to the user initializer. Thought behind caching at login is if the initial call fails then once we have a retry job it would have time to get a successful response before the user navigates to a page that requires the data.